BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -329,6 +329,7 @@ extern NSString *const BOXAPIObjectKeyExtension;
 extern NSString *const BOXAPIObjectKeyIsPackage;
 extern NSString *const BOXAPIObjectKeyAllowedSharedLinkAccessLevels;
 extern NSString *const BOXAPIObjectKeyCollections;
+extern NSString *const BOXAPIObjectKeyCollectionMemberships;
```

Comment:
isn't that supposed to be private fields ?

Reply:
It is public but undocumented, it's ok for all consumers of the Box iOS SDK

Reply:
We need access to the property. I don't see that we have precedence for  a private constant.